Sources/Sentry/SentryCrashIntegration.m

```diff
@@ -248,11 +250,11 @@ + (void)enrichScope:(SentryScope *)scope crashWrapper:(SentryCrashWrapper *)cras
     [deviceData setValue:systemInfo[@"freeMemory"] forKey:SentryDeviceContextFreeMemoryKey];
     [deviceData setValue:systemInfo[@"usableMemory"] forKey:@"usable_memory"];
     [deviceData setValue:systemInfo[@"memorySize"] forKey:@"memory_size"];
-    [deviceData setValue:systemInfo[@"storageSize"] forKey:@"storage_size"];
+    [deviceData setValue:systemInfo[@"totalStorageSize"] forKey:@"storage_size"];
+    [deviceData setValue:systemInfo[@"freeStorageSize"] forKey:@"free_storage"];
     [deviceData setValue:systemInfo[@"bootTime"] forKey:@"boot_time"];
-
-    NSString *locale = [[NSLocale autoupdatingCurrentLocale] objectForKey:NSLocaleIdentifier];
-    [deviceData setValue:locale forKey:LOCALE_KEY];
```

Comment:
I've removed locale from the device context, as we're already sending the culture context which contains the locale as well.

Reply:
`h`: Please don't as locale is indexed and culture context not yet, see https://github.com/getsentry/sentry/issues/30525.

Reply:
Ah, my bad. Restored!